### DISCUSSION TITLE
Radio Group option won't respond on first click

### DISCUSSION THREAD

**Discussion by sukim-twilio** [twilio-labs/paste#1612] [Q&A] [answered]
[summary] user has to click twice to select new options if the first option's fields are filled.
[to regenerate]
1. user clicks `branch`
2. user fills `Branch` field and `Dockerfile` field 
<img width="452" alt="Screen Shot 2021-06-28 at 4 11 23 PM" src="https://user-images.githubusercontent.com/86681213/123842298-7a7b7400-d8c5-11eb-9860-5df5cbcbc7bd.png">

3. user clicks `Image Tag`  ***non-responsive** (displays first pic)
4. user clicks `Image Tag` again, and it resets the field (like 2nd pic)
<img width="446" alt="Screen Shot 2021-06-29 at 10 34 33 AM" src="https://user-images.githubusercontent.com/86681213/123842418-9aab3300-d8c5-11eb-8203-80cfa62efa5a.png">

I think recent refactor was the one causing as the corp is not having an issue. when I brought up this issue to [#eng-admiral](https://twilio.slack.com/archives/C5N73HX0X/p1624922789122000?thread_ts=1624657263.101100&cid=C5N73HX0X), they referred me here as there was something similar reported on another part of the app.

**Comment by richbachman** [comment]
Hi @sukim-twilio. Thanks for posting this. Is there anyway we could see a code sample? Either a link to a repo, or a CodeSandbox would be helpful. Thanks.

**Reply by sukim-twilio** [reply]
I resolved by rendering it within field rather than passing it as a component of a field! thank you for responding!

**Comment by nkrantz** [comment, marked as answer]
Just to recap, the radio button problem was resolved by @sukim-twilio by fixing the rendering method. Thanks!
